conans/client/command.py

```diff
@@ -1711,19 +1711,23 @@ def run(self, *args):
             if exc.code != 0:
                 logger.error(exc)
                 self._user_io.out.error("Exiting with code: %d" % exc.code)
+                sys.stderr.write("Exiting with code: %d" % exc.code)
             ret_code = exc.code
         except ConanInvalidConfiguration as exc:
             ret_code = ERROR_INVALID_CONFIGURATION
             self._user_io.out.error(exc)
+            sys.stderr.write(str(exc))
         except ConanException as exc:
             ret_code = ERROR_GENERAL
             self._user_io.out.error(exc)
+            sys.stderr.write(str(exc))
```

Comment:
This duplicates errors in users output:
```bash
$ conan install zlib/1.2.11@conan/stable -s os=AIX
ERROR: Invalid setting 'AIX' is not a valid 'settings.os' value.
Possible values are ['Android', 'Arduino', 'FreeBSD', 'Linux', 'Macos', 'SunOS', 'Windows', 'WindowsStore', 'iOS', 'tvOS', 'watchOS']
Read "http://docs.conan.io/en/latest/faq/troubleshooting.html#error-invalid-setting"
Invalid setting 'AIX' is not a valid 'settings.os' value.
Possible values are ['Android', 'Arduino', 'FreeBSD', 'Linux', 'Macos', 'SunOS', 'Windows', 'WindowsStore', 'iOS', 'tvOS', 'watchOS']
Read "http://docs.conan.io/en/latest/faq/troubleshooting.html#error-invalid-setting"
```

Reply:
So then we have to do it correctly managing both streams in the conan ouput.

Reply:
I'll update